cmd/config/internal/commands/cmdcreatesetter.go

```diff
@@ -20,14 +20,16 @@ import (
 func NewCreateSetterRunner(parent string) *CreateSetterRunner {
 	r := &CreateSetterRunner{}
 	set := &cobra.Command{
-		Use:     "create-setter DIR NAME VALUE",
-		Args:    cobra.ExactArgs(3),
+		Use:     "create-setter DIR NAME --value VALUE",
```

Comment:
leave the doc as it is since as an argument is still correct

Reply:
Done.